src/allowed_args.cpp

```diff
@@ -678,7 +678,9 @@ static void addBlockCreationOptions(AllowedArgs &allowedArgs)
         .addArg("blockprioritysize=<n>", requiredInt,
             strprintf(_("Set maximum size of high-priority/low-fee transactions in bytes (default: %d)"),
                     DEFAULT_BLOCK_PRIORITY_SIZE))
-        .addArg("blockversion=<n>", requiredInt, _("Generated block version number.  Value must be an integer"));
+        .addArg("blockversion=<n>", requiredInt, _("Generated block version number.  Value must be an integer"))
+        .addDebugArg(
```

Comment:
If this was a tweak we could change it dynamically which would be very useful.

Reply:
done